distributions/Dist_jn1348.py

```diff
@@ -0,0 +1,47 @@
+
+import numpy as np
+from base_distribution import BaseDistribution
+
+class Dist_jn1348(BaseDistribution):
+	def __init__(self):
+		self.f_max = 1
+		self.x_min = np.pi/-2.0
+		self.x_max = np.pi/2.0
+
+
+	def pdf(self, x):
+		"""This is your PDF"""
+		return np.sin(x)
```

Comment:
Hi Professor Cranmer,

I fixed my error, but do I need to submit another pull request because that
button isn't popping up like it did before.

Best,
Jessica Nicholson

On Wed, Apr 13, 2016 at 1:16 PM, Kyle Cranmer <notifications@github.com>
wrote:

> In distributions/Dist_jn1348.py
> <https://github.com/cranmer/intro-exp-phys-II/pull/71#discussion_r59587639>
> :
>
> > @@ -0,0 +1,47 @@
> > +
> > +import numpy as np
> > +from base_distribution import BaseDistribution
> > +
> > +class Dist_jn1348(BaseDistribution):
> > +	def __init__(self):
> > +		self.f_max = 1
> > +		self.x_min = np.pi/-2.0
> > +		self.x_max = np.pi/2.0
> > +
> > +
> > +	def pdf(self, x):
> > +		"""This is your PDF"""
> > +		return np.sin(x)
>
> PDFs must be positive
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/cranmer/intro-exp-phys-II/pull/71/files/393a7e598a5ffc45207c7770fd7de1ce29477687#r59587639>
>